src/main/kotlin/domain/Result.kt

```diff
@@ -0,0 +1,11 @@
+package domain
+
+import domain.player.Players
+
+data class Result(
+    val winners: Players,
+    val losers: Players,
+) {
+    val numOfWinner = winners.list.size
```

Comment:
프로퍼티나 함수의 반환 타입을 명시하면 조금 더 읽기 쉬운 코드가 될 것 같아요🙂
```suggestion
    val numOfWinners: Int = winners.list.size
```